src/main/java/org/zalando/nakadi/domain/ResourceAuthorization.java

```diff
@@ -60,6 +64,15 @@ public ResourceAuthorization(@JsonProperty("admins") final List<AuthorizationAtt
         return writers;
     }
 
+    @JsonIgnore
+    public List<AuthorizationAttribute> getAll() {
+        return Stream.of(
+                Optional.ofNullable(this.readers).orElse(new ArrayList<>()),
+                Optional.ofNullable(this.writers).orElse(new ArrayList<>()),
+                Optional.ofNullable(this.admins).orElse(new ArrayList<>()))
```

Comment:
I think it doesn't make sense to check for null something that is not marked as @Nullable
In Nakadi we keep the not-nullable-by-default practice which allows to keep code clean from unnecessary null checks.

Reply:
Fixed in `de2a148`